operator_claims.go

```diff
@@ -19,21 +19,26 @@ import (
 	"errors"
 	"fmt"
 	"net/url"
+	"strings"
 
 	"github.com/nats-io/nkeys"
 )
 
 // Operator specific claims
 type Operator struct {
-	Identities       []Identity `json:"identity,omitempty"`
-	SigningKeys      StringList `json:"signing_keys,omitempty"`
-	AccountServerURL string     `json:"account_server_url,omitempty"`
+	Identities         []Identity `json:"identity,omitempty"`
+	SigningKeys        StringList `json:"signing_keys,omitempty"`
+	AccountServerURL   string     `json:"account_server_url,omitempty"`
+	OperatorServiceURL string     `json:"operator_service_url,omitempty"`
```

Comment:
If this is where you can connect, should be an array. Even with NGS we could list main URL but then region and CP specific ones as well.

Reply:
Fixed!